bin/compile

```diff
@@ -178,5 +183,10 @@ if test -d $build_dir/bower_components; then
   cp -r $build_dir/bower_components $cache_dir/
 fi
 
+if test -f $build_dir/hooks/after_hook.sh; then
+  status "After Hook detected. Here comes the crocodile..."
```

Comment:
Maybe something like, "After hook detected. Running..." same thing for the before hook.